README.md

```diff
@@ -1 +1,83 @@
-# kf-task-fhir-etl
\ No newline at end of file
+# Kids First FHIR ETL Task Service
+
+The Kids First FHIR ETL Task Service, built off of the [Kids First Data Ingest Library](https://github.com/kids-first/kf-lib-data-ingest), is a Python wrapper application, which
+
+- Extracts tables from the KF Dataservice DB;
+- Transforms the extracted tabular data to FHIR resources in JSON; and
+- Loads the transformed records into a Kids First FHIR Service.
+
+## Quickstart
```

Comment:
Leaving this here as a reminder for us to create a ticket later. This can be done in another PR, but would be great if we had a docker-compose.yaml so that a developer can get this up and running locally fairly quickly. We could use the free HAPI FHIR server for our FHIR service

Reply:
Yes, I will do that with #7.